pontoon/contributors/views.py

```diff
@@ -278,6 +279,13 @@ def mark_all_notifications_as_read(request):
     """Mark all notifications of the currently logged in user as read"""
     request.user.notifications.mark_all_as_read()
 
+    utm_source = request.GET.get("utm_source")
```

Comment:
I think we should collect this ux action on all calls, and add `utm_source` as data.